### DISCUSSION TITLE
Continue reading ... in blog post index site

### DISCUSSION THREAD

**Discussion by max-hammer** [squidfunk/mkdocs-material#7848] [Q&A] [answered]
Hi there

I've created a mkdocs blog post but i cant see this "Continue Reading" button. How can i enable them?

![screenshot](https://github.com/user-attachments/assets/2afde74e-7cdf-46ac-9ccc-1fe2c2f05a3e)

can someone provide a simple snippet (if needed with html and css)?

Greedings

**Comment by niclasheinz** [comment, marked as answer]
Hello @opensource-coder909 

You can just add this line where you want the ‘Continue reading’ text.
```
<!-- more -->
```
and it looks like this:
![image](https://github.com/user-attachments/assets/cc7e7a91-a270-4951-8508-4580bfe1aced)

For more information about the blog feature you can visit the [Blog Tutorial ](https://squidfunk.github.io/mkdocs-material/tutorials/blogs/basic/#basic-blogs)

EDIT: Because of the setting `<!-- more -->` HTML and CSS is not needed

**Reply by max-hammer** [reply]
Thx it works
